group_layers/group-layers/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 group = 'com.esri.samples'
 
 ext {
-    arcgisVersion = '100.9.0'
+    arcgisVersion = '100.10.0-2950'
```

Comment:
might get a merge conflict in this file with the updated gradle/jfrog changes: or hopefully it just updates no problem!